app/include/user_modules.h

```diff
@@ -27,6 +27,7 @@
 #define LUA_USE_MODULES_HTTP
 //#define LUA_USE_MODULES_HX711
 #define LUA_USE_MODULES_I2C
+#define LUA_USE_MODULES_MDNS
```

Comment:
I'd rather have this disabled by default even though #872 is still waiting for a resolution.